.appveyor.yml

```diff
@@ -13,6 +13,7 @@ build: off
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
   - npm i -g npm
+  - npm i
   - npm ci
```

Comment:
Are we facing issues with `npm ci`? Also, if we're doing `npm i` then we probably don't need to run `npm ci`.

Reply:
Just people not updating package-locks, it's a safeguard.